spec/models/bookmark_search_form_exclusion_filters_spec.rb

```diff
@@ -117,19 +117,19 @@
         expect(search.search_results).not_to include(excluded_bookmark)
       end
 
-      it "should exclude bookmarks tagged with a canonical tag given that tag's synonym" do
-        excluded_bookmark.update(tag_string: "Complete")
-        update_and_refresh_indexes("bookmark")
+      # it "should exclude bookmarks tagged with a canonical tag given that tag's synonym" do
```

Comment:
I think we might want to either a) remove this or b) add a note about why it's commented out, just so no one comes across this later and thinks "Oh, a failing test to fix!" or "Oh, behavior I should implement!"

Reply:
I can remove it. Should I remove the other commented out tests?

Reply:
If they're tests for things we won't want to implement, yep, please do!